templates/emails/email-order-items.php

```diff
@@ -43,8 +43,6 @@
 				// allow other plugins to add additional product information here
 				do_action( 'woocommerce_order_item_meta_start', $item_id, $item, $order, $plain_text );
 
-				wc_display_item_meta( $item );
```

Comment:
This needs a different approach. This method outputs other non-attribute data too e.g. if using product addons extension, it will list other selected values which won't be included in the title. We should instead modify this method to skip over attribute values which I believe have a `pa_` prefix.